DEPENDENCY-LICENSES

```diff
@@ -531,7 +531,7 @@ List of third-party dependencies grouped by their license type.
 
     Apache License, Version 2.0, GNU General Public License, version 2
 
-        * RocksDB JNI (org.rocksdb:rocksdbjni:5.18.4 - http://rocksdb.org/)
+        * RocksDB JNI (org.rocksdb:rocksdbjni:6.27.3 - http://rocksdb.org/)
```

Comment:
As per the build error, this line needs to be changed to remove the trailing slash:
* RocksDB JNI (org.rocksdb:rocksdbjni:6.27.3 - http://rocksdb.org/)
should be 
* RocksDB JNI (org.rocksdb:rocksdbjni:6.27.3 - http://rocksdb.org)